telemetry/url.go

```diff
@@ -0,0 +1,15 @@
+package telemetry
+
+import (
+	"context"
+
+	"go.opentelemetry.io/otel/trace"
+)
+
+func URLForTrace(ctx context.Context) (string, bool) {
+	if !configuredCloudTelemetry {
+		return "", false
+	}
+	url := "https://dagger.cloud/traces/" + trace.SpanContextFromContext(ctx).TraceID().String()
```

Comment:
The Cloud URL requires the org name: `https://dagger.cloud/<ORG>/traces/<TRACE ID>`

Unfortunately, we don't have that readily available, but that's being fixed:

- When using `dagger login` https://github.com/dagger/dagger/pull/7125, there will be an `org.Name` available
- When using a `DAGGER_CLOUD_TOKEN`, there's a new token format that allows deriving the org name from the token (`dag_<org name>_secret`). Coming today
  - If using an old token format, we should tell the user a generic "Visit https://dagger.cloud -- blah blah -- rotate your token if you want to see the trace URL from the CLI"

Reply:
Mm, I figured this would be required - that said, it does still "seem" to land on the right page interestingly.

Reply:
> Mm, I figured this would be required - that said, it does still "seem" to land on the right page interestingly.

Yeah, AFAIK if no org is explicitly passed, then it just redirects to "current" org (current being the last one you used). So it does work if the conditions are alright, but it's not super dependable